File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.java
class Solution {
    public int[] runningSum(int[] nums) {
        int[] res = new int[nums.length];
        int run = 0;
        for(int i = 0; i < nums.length; i++){
            run += nums[i];
            res[i] = run;
        }
        
        return res;
    }
}

File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var runningSum = function(nums) {
    let sum = 0;
    
    for(let i = 1; i < nums.length; i++){
        // console.log(num);
        nums[i] = nums[i] + nums[i - 1];
    }
    
    return nums;
};